deps/build.jl

```diff
@@ -1,14 +1,17 @@
 using BinDeps
 
-@BinDeps.setup
-
 using Compat
+import Compat.Libdl
+import Compat.Sys
+import Compat.Pkg
```

Comment:
this doesn't exist

Reply:
i cannot follow, as it does (here locally in 0.51.0).
```
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.7.0-DEV.3677 (2018-02-01 16:36 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit ecb8b56* (2 days old master)
|__/                   |  x86_64-linux-gnu

julia> using Compat

help?>  Compat.Libdl
  Interface to libdl. Provides dynamic linking support.


julia> import Compat.Libdl

julia> 
```

Reply:
I think Tony is referring to `Compat.Pkg`. That was introduced in Compat v0.52.0, which postdates this PR (tagged today).

Reply:
Strange, still 0.51.0 here and 
```
               _
   _       _ _(_)_     |  A fresh approach to technical computing
  (_)     | (_) (_)    |  Documentation: https://docs.julialang.org
   _ _   _| |_  __ _   |  Type "?help" for help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 0.7.0-DEV.3677 (2018-02-01 16:36 UTC)
 _/ |\__'_|_|_|\__'_|  |  Commit ecb8b56* (2 days old master)
|__/                   |  x86_64-linux-gnu

julia> using Compat

julia> import Compat.Pkg
WARNING: importing deprecated binding Base.Pkg into Compat.

julia> 
```

Reply:
same warning in 0.50.0